_includes/manuals/nightly/3.3.1_rpm_packages.md

```diff
@@ -1,44 +1,12 @@
 
 Foreman is packaged for the following RPM based distributions:
 
-* RHEL and derivatives, version 7
 * RHEL and derivatives, version 8
 
 **Note:** The RPM packages are not tested on Scientific Linux or Oracle Linux. The Foreman installation on Scientific Linux or Oracle Linux may or may not work.
```

Comment:
Scientific Linux never released version 8 so the mention can be dropped or replaced by Rocky Linux
```suggestion
**Note:** The RPM packages are not tested on Rocky Linux or Oracle Linux. The Foreman installation on Rocky Linux or Oracle Linux may or may not work.
```